### PR TITLE
Add the cover picture behavior

### PR DESCRIPTION
**Pull request by ale-rt**
This is built upon #10

### CODE REVIEW

**Comment by ale-rt** [comment]
QA instructions:

1. reinstall the package (no upgrade steps yet)
2. Create some persone in the site root
3. Create a Publication in the site root
4. Crete a Volume in the Publication
5. Four different "Review" types can be added: you can upload to them a cover picture

File: src/recensio/plone/behaviors/cover_picture.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    def __init__(self, context):
+        self.context = context
+
+    @property
+    def coverPicture(self):
+        return self.context.coverPicture
+
+    @coverPicture.setter
+    def coverPicture(self, value):
+        self.context.coverPicture = value
```
Comment (thet) [inline src/recensio/plone/behaviors/cover_picture.py:32]:
```suggestion
```
I think we do not need a factory for this behavior, right?

File: src/recensio/plone/behaviors/configure.zcml [hunk 2]
```diff
@@ -11,4 +11,12 @@
       for="plone.dexterity.interfaces.IDexterityContent"
       />
 
+  <plone:behavior
+      name="recensio.cover_picture"
+      title="Cover Picture Behavior"
+      factory=".cover_picture.CoverPicture"
+      provides=".cover_picture.ICoverPicture"
+      for="plone.dexterity.interfaces.IDexterityContent"
```
Comment (thet) [inline src/recensio/plone/behaviors/configure.zcml:19]:
```suggestion
```

File: src/recensio/plone/behaviors/configure.zcml [hunk 3]
```diff
@@ -11,4 +11,12 @@
       for="plone.dexterity.interfaces.IDexterityContent"
       />
 
+  <plone:behavior
+      name="recensio.cover_picture"
+      title="Cover Picture Behavior"
+      factory=".cover_picture.CoverPicture"
```
Comment (thet) [inline src/recensio/plone/behaviors/configure.zcml:17]:
```suggestion
```
I think we do not need a factory for this behavior, right?

**Review by thet — CHANGES_REQUESTED** [review]
As with #10, I‌ think we can leave out the behavior factory unless you plan to extend it in the future.

Otherwise, LGTM

**Pull request merged**